src/util/heuristic.jl

```diff
@@ -58,28 +58,28 @@ function random_repair(network_data; repair_component_types=["gen", "branch", "s
 
 end
 
+# not developed function #
+# ""
+# function largest_item_repair(network_data)
 
-""
-function largest_item_repair(network_data)
-
-    networks = []
-    if haskey(network_data, "nw")
-        for n in keys(network_data["nw"])
-            push!(networks, parse(Int, n))
-        end
-    else
-        # TODO push error for non-multinetwork
-    end
+#     networks = []
+#     if haskey(network_data, "nw")
+#         for n in keys(network_data["nw"])
+#             push!(networks, parse(Int, n))
+#         end
+#     else
+#         # TODO push error for non-multinetwork
+#     end
 
-    total_repairs = 0 #count damaged items
+#     total_repairs = 0 #count damaged items
 
-    repair_periods = length(networks)
+#     repair_periods = length(networks)
 
-    repairs_per_period = ceil(Int, total_repairs/repair_periods)
```

Comment:
Same issue as `calc_cumulative_repairs_per_period`, better to look up how many repairs are allowed in this period from the data model and execute exactly that many.

Reply:
done